src/features/calendar-events/index.js

```diff
@@ -1,33 +1,65 @@
-import React from 'react'
-import { Calendar } from 'react-big-calendar'
+import React, { useState } from 'react'
 
 import config from './calendarConfig'
+import {
+  CalendarEventsContainer,
+  CurrentEventsNumber,
+  StyledCalendar,
+} from './styled-components'
 
-import 'react-big-calendar/lib/css/react-big-calendar.css'
+const {
+  defaultDate,
+  drilldownView,
+  defaultView,
+  localizer,
+  selectable,
+  resizable,
+} = config
 
 const CalendarRBC = () => {
-  const {
-    defaultDate,
-    drilldownView,
-    defaultView,
-    events,
-    localizer,
-    resizable,
-    style,
-  } = config
+  const [calendarEvents, setCalendarEvents] = useState([
+    {
+      start: new Date('2019, 12, 31'),
+      end: new Date('2020, 01, 01'),
+      title: 'Happy New Year',
+      allDay: true,
+    },
+  ])
+
+  const handleSelect = ({ start, end }) => {
+    if (start.getDate() >= defaultDate.getDate()) {
+      // eslint-disable-next-line no-alert
```

Comment:
Why we use eslint if you disable it

Reply:
Unexpected prompt, idk

Reply:
**Examples of incorrect code for this rule:**
`/*eslint no-alert: "error"*/
alert("here!");
confirm("Are you sure?");
prompt("What's your name?", "John Doe");`

**Examples of correct code for this rule:**
`/*eslint no-alert: "error"*/
customAlert("Something happened!");
customConfirm("Are you sure?");
customPrompt("Who are you?");
function foo() {
    var alert = myCustomLib.customAlert;
    alert();
}`

`https://eslint.org/docs/rules/no-alert`